mod/Jailbreak.Warden/Global/WardenPeaceBehaviour.cs

```diff
@@ -0,0 +1,80 @@
+﻿using CounterStrikeSharp.API;
+using CounterStrikeSharp.API.Core;
+using Jailbreak.Public.Behaviors;
+using Jailbreak.Public.Generic;
+using Jailbreak.Public.Mod.Plugin;
+using Jailbreak.Public.Mod.Warden;
+
+namespace Jailbreak.Warden.Global;
+
+public class WardenPeaceBehaviour : IPluginBehavior, IWardenPeaceService
+{
+
+    private readonly IWardenService _wardenService;
+    private readonly ICoroutines _coroutines;
+    private readonly IEventsService _eventsService;
+
+    private readonly float _muteTime = 10.0f;
```

Comment:
If possible distinguish this variable from https://github.com/edgegamers/Jailbreak/pull/69/files#diff-e81c196d0b0e42fe347b9e3ee80c463e42c9aa74b8baec3127ed11383c01feaeR16

Reply:
Done